keylime/cloud_verifier_tornado.py

```diff
@@ -105,6 +105,23 @@
 }
 
 
+# TODO(kaifeng) Move this to db as well as other db objects
+def _from_db_obj(agent_db_obj):
+    fields = ['agent_id', 'v', 'ip', 'port',
+              'operational_state', 'public_key',
+              'tpm_policy', 'vtpm_policy', 'meta_data',
+              'ima_whitelist', 'revocation_key',
+              'tpm_version',
+              'accept_tpm_hash_algs',
+              'accept_tpm_encryption_algs',
+              'accept_tpm_signing_algs',
+              'hash_alg', 'enc_alg', 'sign_alg']
+    agent_dict = {}
+    for field in fields:
+        agent_dict[field] = getattr(agent_db_obj, field)
+    return agent_dict
```

Comment:
Looks good, do you think we could benefit from a try / except here..perhaps check for KeyErrors?

Reply:
I don't worry about this, if it's a db object, it should have those fields, except there are other code path I am unaware of. To be defensive, we could add a default value though, it will not raise no matter a field exists or not.